package.json

```diff
@@ -11,6 +11,14 @@
       "import": "./index.mjs",
       "require": "./index.js"
     },
+    "./dom-to-react": {
+      "types": "./lib/dom-to-react.d.ts",
+      "default": "./lib/dom-to-react.js"
```

Comment:
It's not possible, the "exports" field in a package.json limits the things you can import. So right now, only `import ... from "html-react-parser"` and `import ... from "html-react-parser/package.json"` are possible.

We can also do:

```
"./lib/*": {
  "types": "./lib/*.d.ts",
  "default": "./lib/*.js"
}
```

This would allow importing any file in lib.

Reply:
Gotcha, are you using CommonJS or ESM?

Reply:
I'm using ESM.
However this will not work in Node >12.7, both CJS and ESM:

```
> require('html-react-parser/lib/dom-to-react.js')
Uncaught:
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './lib/dom-to-react.js' is not defined by "exports" in /<root>/node_modules/html-react-parser/package.json
    at __node_internal_captureLargerStackTrace (node:internal/errors:484:5)
    at new NodeError (node:internal/errors:393:5)
    at throwExportsNotFound (node:internal/modules/esm/resolve:358:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:668:3)
    at resolveExports (node:internal/modules/cjs/loader:529:36)
    at Module._findPath (node:internal/modules/cjs/loader:569:31)
    at Module._resolveFilename (node:internal/modules/cjs/loader:981:27)
    at Module._load (node:internal/modules/cjs/loader:841:27)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}
```

Reply:
Makes sense, let's go with:

```
"./lib/*": {
  "types": "./lib/*.d.ts",
  "default": "./lib/*.js"
}
```